### ISSUE TITLE
MRT app path not correct for version capture

### ISSUE THREAD

**Issue by apizz** [munkireport/softwareupdate#8]
Related code: https://github.com/munkireport/softwareupdate/blob/master/scripts/softwareupdate#L161-L176

Currently the defined path is `/System/Library/CoreServices/MRT.app`, but on newer macOS versions the path is actually `/Library/Apple/System/Library/CoreServices/MRT.app`, and as a result isn't returning data.

Rather than hardcode these paths, I propose using `mdfind` to get the MRT path for the particular system based on its Bundle Identifier (`com.apple.MRT`).  This is similar to what we do now currently in our MDM.

Example command & output:
```
mdfind kMDItemCFBundleIdenterifier = "com.apple.MRT"
/Library/Apple/System/Library/CoreServices/MRT.app
```

So something like:
`defaults read $(mdfind kMDItemCFBundleIdenterifier = "com.apple.MRT")/Contents/Info.plist CFBundleShortVersionString)`

**Comment by tuxudo** [comment]
The softwareupdate script has a few problems with macOS 11. Planning on doing a slight overhaul to it in the coming week.

**Comment by tuxudo** [comment]
Should be resolved with version v1.8 of the module
